platform_versioned_docs/version-24.1.1/getting-started/quickstart-demo/pipeline-optimization.mdx

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Its content lives inside the comm-showcase page, and will live in the full RNASeq guide as well. I need to update it slightly so that it's not referencing a comm showcase run for the RNASeq guide, so I'm keeping the page here for now without including it in the sidebar.